config/earlyapp_gst.target

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=earlyapp target
```

Comment:
Explicitly mention that this target file is for the Early App GStreamer service in description.